docs/samples/Microsoft.ML.Samples.StaticPipe/DatasetCreator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Samples.StaticPipe
+{
+    public static class DatasetCreator
+    {
+        public static (string trainPath, string testPath) CreateRegressionDataset()
+        {
+            // creating a small sample dataset, and writting it to file
+            string trainDataPath = @"c:\temp\RegressionTrainDataset.txt";
```

Comment:
We shouldn't be writing to `C:\temp`. Instead we should write into the repo's `bin/obj` directory.

Reply:
Some people may not have a `C` drive. Some people might not be using Windows.

---
In reply to: [221757977](https://github.com/dotnet/machinelearning/pull/1105#discussion_r221757977) [](ancestors = 221757977)

Reply:
Eric, I am leaving it just as a file in the current folder, since before merging, we should decide whether to package the datasets we have in a nuget, or package files we create, like this one for every task ..  

---
In reply to: [221758066](https://github.com/dotnet/machinelearning/pull/1105#discussion_r221758066) [](ancestors = 221758066,221757977)